packages/service-clients/odsp-client/api-report/odsp-client.alpha.api.md

```diff
@@ -17,13 +17,13 @@ export interface IOdspTokenProvider {
 export class OdspClient {
     constructor(properties: OdspClientProps);
     // (undocumented)
-    createContainer<T extends ContainerSchema>(containerSchema: T): Promise<{
```

Comment:
Note: the changes in this package will not be checked in. They were made just to illustrate how a service client implementation could opt into tree-only mode.